### ISSUE TITLE
Remove `onchange` attribute from #fed

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/veterans-employment-center#307]
Related to #305, #306.

508 report flagged this as a _Potential violation_. Error message:

> WCAG 2, 3.2.2 - On Input (H32, H84, G13), Level A, Make sure that changes of context are announced if occur when select control value changes.

Offending code:

`<select name="fed" id="fed" class="select" onchange="">`

It doesn't look like this event attribute is doing anything, so it's safe to remove it from the markup.

**Comment by webinista** [comment]
Old enough that it's probably irrelevant. Closing.
